### ISSUE TITLE
Find a home on fec.gov for accessing the Record archive

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2839]
**What we're after**
It's a pain point to find the older archived Record newsletters on the transition page, especially given that the Record lives on fec.gov. This ticket is for figuring out an easier short-term solution for getting to the archive.

**Background**
Right now the index of older Record newsletters lives at https://transition.fec.gov/pages/fecrecord/fecrecord.shtml. To get there, you click a link on https://transition.fec.gov/. Would like to work with design or front end team to find a way to put that link on the main updates page at https://www.fec.gov/updates/, where the Record now lives.

**To do**
- [x] Speak with Info Division, design team, and front end team for ideas
- [x] Determine if we want to move that index page into the CMS. (The newsletter PDF files are on the CMS) - YES WE DO
- [x] Create the index page in the CMS as a resource page in Updates
- [x] Draft wording to link to the new page
- [x] Get stakeholder ASD approval for both.
- [x] Publish new page (second pair of eyes)
- [x] Add old Record archive pages at https://transition.fec.gov/pages/fecrecord/fecrecord.shtml and https://transition.fec.gov/pages/record.shtml to the redirect spreadsheet to put in redirect to new page.
- [x] Put in ticket for front end implementation work to update the Updates message once we ~determine best approach~ have the index page ready to go.

**Comment by dorothyyeager** [comment]
After speaking with the Deputy ASD for publications and @JonellaCulmer, we all like the idea of using the resource page template to create a page in Wagtail that will live under updates and be linked from the updates dialog box that explains what's on the page. Will finalize a draft page for review by the Info Division ASD, and work on publishing that in this sprint. Will also draft a message to use for linking the new page and get Info Div's sign off on that. I've updated the completion criteria with all of these items.

**Comment by dorothyyeager** [comment]
Work continues on the draft page for the archive.

**Comment by dorothyyeager** [comment]
I've sent the draft page and wording for the link to the Information Division ASD for approval.

**Comment by dorothyyeager** [comment]
We've published the archive of older Record articles from 1975-2004 at https://www.fec.gov/updates/record-archive-1975-2004/. I've updated the page at https://transition.fec.gov/ to link to this page instead of the older version of the archive. I've also added the new page/old page to our redirect spreadsheet and have put in a ticket to add the link to the new archive to the "About Latest Updates" information box. Work will continue on that in #3122 ; this issue can be closed.
